src/js/helpers/events.js

```diff
@@ -15,8 +15,20 @@ export function handleEvent(eventObj, clickHandlers) {
     eventCode === 'txSpeedUp' ||
     eventCode === 'txCancel'
 
-  // If not a server event then log it
-  !serverEvent && lib.logEvent(eventObj)
+  let eventToLog = { ...eventObj }
+
+  // If dealing with a custom notification the logged event
+  // should have it's event and category code changed
+  if (categoryCode === 'userInitiatedNotify') {
+    eventToLog = {
+      ...eventToLog,
+      categoryCode: 'custom',
+      eventCode: 'txNotification'
```

Comment:
let's change this to just 'notification' since it make not actually involve a transaction.